Tools-Override/FrameworkTargeting.targets

```diff
@@ -4,10 +4,10 @@
   <ItemGroup>
     <TargetingPackDirs Include="$(RefPath)" />
     <AdditionalReferencePaths Include="@(TargetingPackDirs)" />
+    <Reference Condition="($(NuGetTargetMoniker.StartsWith('.NETStandard')) OR '$(IsReferenceAssembly)' != 'true') AND Exists('$(RefPath)netstandard.dll') AND '$(MSBuildProjectExtension)' != '.pkgproj'" Include="netstandard" />
```

Comment:
Better but I think I would tweak it along the lines of:

```
  <PropertyGroup Condition="'$(IncludeDefaultReferences)' == ''">
    <IncludeDefaultReferences Condition="'$(MSBuildProjectExtension)' == '.csproj'">true</IncludeDefaultReferences>
    <IncludeDefaultReferences Condition="'$(MSBuildProjectExtension)' == '.vbproj'">true</IncludeDefaultReferences>
  </PropertyGroup>

  <ItemGroup Condition="'$(IncludeDefaultReferences)' =='true'" >
    <!-- netstandard is a default reference whenever building for NETStandard or building an implementation assembly -->
    <DefaultReference Condition="$(NuGetTargetMoniker.StartsWith('.NETStandard') OR '$(IsReferenceAssembly)' != 'true'"
                       Include="netstandard" />

    <!-- Include default references when specified and they exist -->
    <Reference Condition="'Exists('$(RefPath)%(Identity).dll')" Include="@(DefaultReference)" />
  </ItemGroup>
```

Also I think this is the right file as I want all of targetingpacks.props to go into here.